core/src/main/java/cn/edu/tsinghua/iot/benchmark/conf/Config.java

```diff
@@ -529,14 +537,12 @@ void initSensorCodes() {
     }
   }
 
-  /** Generate Probabilities according to proportion(e.g. 1:1:1:1:1:1) */
+  /** Generate Probabilities according to proportion(e.g. 1:1:1:1:1:1:1:1:1:1) */
```

Comment:
We can keep consistent with default value in config here. 1:1:1:1:1:1:0:0:0:0

Reply:
OK